src/recon_buildblock/IterativeReconstruction.cxx

```diff
@@ -397,16 +397,16 @@ reconstruct()
 {
   this->start_timers();
 
-  this->target_data_sptr.reset(this->get_initial_data_ptr());
-  if (this->set_up(this->target_data_sptr) == Succeeded::no)
+  shared_ptr<TargetT > target_data_sptr(this->get_initial_data_ptr());
```

Comment:
why do we do this? It's normally using `Reconstruct::target_data_sptr`. Changing this to use a local variable is scary.

Reply:
I tried to track this down and goes back to 13/07/2016. It was part of two commits 2d4e3067f32542a5e8ec297260ece44b832c7b76 and efed646d94b4ab030366f6c7170146c1126b5f12 to be able to reconstruct without output.

Reply:
I cannot remember how this didn't come to UCL/STIR as back then we had accepted this, as part of the scatter correction

Reply:
I could revert them both if you're concerned.